dev/ProgressRing/ProgressRing.cpp

```diff
@@ -119,3 +205,13 @@ void ProgressRing::UpdateRing()
 
     UpdateSegment();
 }
+
+winrt::Size ComputeCircleSize(double thickness, double actualWidth)
```

Comment:
>Size [](start = 7, length = 4)

I still think you should change this to return a float, and rename to GetCircleRadius